core/index-api.js

```diff
@@ -887,7 +888,8 @@ let API_ROUTES = {
 						}
 
 						await ctx.store.tx(tx._id, {
-							error: errorCode, 
+							errorCode: errorCode,
```

Comment:
don't see errorCode definintion. Should it be tx.errorCode?

Reply:
No, it should not. Definition was there, with this change I just fixed its using.